packages/integrations/lit/server.js

```diff
@@ -1,6 +1,6 @@
-import './server-shim.js';
 import { LitElementRenderer } from '@lit-labs/ssr/lib/lit-element-renderer.js';
 import * as parse5 from 'parse5';
+import './server-shim.js';
```

Comment:
I think we need the `import './server-shim.js';` to be first as it's side-effectful and presumably affects how `@lit-labs/ssr/lib/lit-element-renderer.js` is loaded, so the order matters.

Reply:
They should honestly be in separate group if the order is important, any formatter would make this mistake

Reply:
Yeah putting a space between the imports would be fine by me too. I guess if biome wants to be more cautious about this automatically, plain `import '...'` ordering should be left as-is.